docs/index.md

```diff
@@ -39,6 +39,7 @@ Usage of rtlamr:
   -format=plain: format to write log messages in: plain, csv, json, xml or gob
   -gobunsafe=false: allow gob output to stdout
   -logfile=/dev/stdout: log statement dump file
+  -logappend=false: append to, instead of replacing, logfile at startup
```

Comment:
Please revert this change, `docs/index.md` is kept in sync with `README.md` manually and I will update it at the next release.